packages/ipfs/README.md

```diff
@@ -0,0 +1,26 @@
+# @windingtree/org.id-ipfs
+ORGiD IPFS library
+
+## Documentation
+
```

Comment:
For now, just add here: TBD 
In the PR description should be mentioned that this package is requiring documentation.
Also, you have to add a task to the backlog "Create documentation for package..."